src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -399,52 +406,57 @@ def on_request(self, request, **kwargs):
 
 # pylint: disable=unused-argument
 def deploy_arm_template_at_subscription_scope(cmd,
-                                              template_file=None, template_uri=None, parameters=None,
+                                              template_file=None, template_uri=None, template_spec=None, parameters=None,
```

Comment:
Could the new parameters be added to the end of all parameters? This ensures that the usage of the caller is not affected